shared/src/api/extension/flatExtensionApi.ts

```diff
@@ -103,6 +134,18 @@ export const initNewExtensionAPI = (
                     )
                 )
             ),
+
+        // Language
+        getHover: (textParameters: TextDocumentPositionParams) => {
+            const document = textDocuments.get(textParameters.textDocument.uri)
+
+            const invokeProvider = (
+                provider: sourcegraph.HoverProvider
+            ): sourcegraph.ProviderResult<sourcegraph.Badged<Hover>> =>
+                provider.provideHover(document, toPosition(textParameters.position))
+
+            return proxySubscribable(callProviders(state.hoverProviders, document, invokeProvider, mergeHoverResults))
```

Comment:
With `position` extracted above, could we just have `invokeProvider` declared inline?

```suggestion
            return proxySubscribable(callProviders(state.hoverProviders, document, provider => provider.provideHover(document, position), mergeHoverResults))
```